core/src/test/scala/org/apache/spark/sql/crossdata/catalog/JdbcCatalogIT.scala

```diff
@@ -51,12 +50,38 @@ class MySQLCatalogIT extends SharedXDContextTest with MySQLCatalogConstants {
 
   }
 
+
+  it should "persist a table with catalog and partitionColumns without schema in Jdbc" in {
+
+    val tableIdentifier = Seq(Database, TableName)
+    val crossdataTable = CrossdataTable(TableName, Option(Database), None, "org.apache.spark.sql.json", Array[String](), OptsJSON)
+    xdContext.catalog.persistTable(crossdataTable)
+    xdContext.catalog.tableExists(tableIdentifier) shouldBe true
+
+    val df = xdContext.sql(s"SELECT * FROM $Database.$TableName")
+    df shouldBe a[XDDataFrame]
+
+  }
+
+  it should "persist a table with catalog and partitionColumns with multiple subdocuments as schema in JDBC" in {
+    xdContext.catalog.dropAllTables()
+    val tableIdentifier = Seq(Database, TableName)
+    val crossdataTable = CrossdataTable(TableName, Option(Database), Option(ColumnsWithSubColumns), "org.apache.spark.sql.json", Array[String](), OptsJSON)
+    xdContext.catalog.persistTable(crossdataTable)
+    xdContext.catalog.tableExists(tableIdentifier) shouldBe true
```

Comment:
It 'd be better to unregister the table in order to test the metadata serialized in the persisted catalog.